spec/schedulers/AsapScheduler-spec.ts

```diff
@@ -40,6 +40,57 @@ describe('Scheduler.asap', () => {
     sandbox.restore();
   });
 
+  it('should recycle the interval for recursively scheduled actions with the same delay', () => {
```

Comment:
Nit: This should probably say "reuse" rather than "recycle"... To me, "recycle" means you're tearing it down and creating another. Which is the opposite of the meaning here.